examples/light-switch-app/nrfconnect/main/include/BindingHandler.h

```diff
@@ -0,0 +1,52 @@
+/*
+ *
+ *    Copyright (c) 2022 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include <app-common/zap-generated/ids/Clusters.h>
+#include <app-common/zap-generated/ids/Commands.h>
+#include <app/CommandSender.h>
+#include <app/clusters/bindings/BindingManager.h>
+#include <controller/InvokeInteraction.h>
+#include <platform/CHIPDeviceLayer.h>
+
+class BindingHandler
+{
+public:
+    static void Init();
+    static void SwitchWorkerHandler(intptr_t);
+
+    struct BindingData
+    {
+        chip::EndpointId endpointId = 1;
```

Comment:
I think we should not hardcode this id here, as this information is hidden from user. If someone would take our sample and want to change the endpoint he would change only kLightSwitchEndpointId in the AppTask and not change that one, so binding would not work. Try to make those two values be assigned by the one common const.